pkg/tests/api/alerting/api_alertmanager_test.go

```diff
@@ -230,24 +251,71 @@ func TestAlertRuleCRUD(t *testing.T) {
                   "namespace_uid":"nsuid",
                   "namespace_id":1,
                   "rule_group":"arulegroup",
-                  "no_data_state":"",
-                  "exec_err_state":""
+                  "no_data_state":"Alerting",
+                  "exec_err_state":"KeepLastState"
                }
             }
          ]
       }
    ]
-}`, rulesNamespaceWithoutVariableValues(t, b))
+}`, rulesNamespaceWithoutVariableValues(t, b, ruleUIDs))
```

Comment:
I think it's better semantically if make this function return the UIDs rather than passing the argument